test/unit/lib/plugins/aws/package/compile/events/api-gateway/lib/rest-api.test.js

```diff
@@ -148,6 +151,30 @@ describe('#compileRestApi()', () => {
       Properties: {
         BinaryMediaTypes: ['*/*'],
         DisableExecuteApiEndpoint: undefined,
+        Description: undefined,
+        EndpointConfiguration: {
+          Types: ['EDGE'],
+        },
+        Name: 'dev-new-service',
+        Policy: '',
+      },
+    });
+  });
+
+  it('should set description if defined at the apiGateway provider config level', () => {
```

Comment:
We should not introduce new tests based on the "legacy" approach - all new tests should be written with `runServerless` utility - please see the tests at the bottom for an example

Reply:
Thanks for taking your time to kind review.
I should be more careful to read the doc…
I updated new test with using runServerless. Nice util!